website/docs/r/cloudant_database.html.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: "ibm"
+page_title: "IBM : cloudant_database"
+description: |-
+  Manages cloudant_database.
+subcategory: "Cloudant"
+---
+
+# ibm\_cloudant_database
+
+Provides a resource for cloudant_database. This allows cloudant_database to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_cloudant_database" "cloudant_database" {
+  instance_crn  = var.instance_crn
+  db            = var.db_name
+}  
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `instance_crn` - (Required, string) Path parameter to specify the cloudant instance CRN.
+* `db` - (Required, Forces new resource, string) Path parameter to specify the database name.
```

Comment:
The arguments/attributes should be placed in alphabatical order..same for all docs